lte/cloud/go/plugin/mconfig.go

```diff
@@ -126,6 +126,7 @@ func (s *LteMconfigBuilderServicer) Build(
 			AttachedEnodebTacs:       getEnodebTacs(enbConfigsBySerial),
 			DnsPrimary:               gwEpc.DNSPrimary,
 			DnsSecondary:             gwEpc.DNSSecondary,
+			NatEnabled:               swag.BoolValue(gwEpc.NatEnabled),
```

Comment:
Do we actually care about the gw EPC config anymore, or can we read this off the network EPC config's IP allocation mode? e.g. is it valid to enable NAT but use the DHCP allocator? Should we set this based on the IP allocation mode? Or is NAT enabled/disabled orthogonal to IP allocation

Reply:
There are cases when we want IP_POOL allocator and Non-NAT mode.
DHCP and NAT does not make much sense, but I do not want to stop customer from setting this configuration.
So basically these two arguments should be decoupled.